### PR TITLE
[system health] Add implementation description for system health hld 

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
Add more description for system health hld according to the implementation.

### CODE REVIEW

File: doc/system_health_monitoring/system-health-HLD.md [hunk 1]
```diff
@@ -117,11 +164,23 @@ If monit service is not avalaible, will consider system in fault condition.
...
+System health service will populate system health data to STATE db. A new table "SYSTEM_HEALTH_INFO" will be created to STATE db.
+
+	; Defines information for a system health
+	key                     = SYSTEM_HEALTH_INFO             ; health information for the switch 
+	; field                 = value
+	summary                 = STRING                         ; summary status for the switch
+	<item_name>             = STRING                         ; an entry for a service or device
 
-## 3. Platform API and PMON related change to support this new service
+We store items to db only if it is abnormal.
```
Comment (keboliu) [inline doc/system_health_monitoring/system-health-HLD.md:181]: can you add an example DB info table here?

File: doc/system_health_monitoring/system-health-HLD.md [hunk 2]
```diff
@@ -68,23 +68,67 @@ By default any above services or file systems is not in good status will be cons
...
+
+For example, there is a python script "my_external_checker.py", and its output is like:
+
+```
+MyCategory
+device1:OK
+device2:device2 is out of power
+```
+
+The configuration could be:
```
Comment (keboliu) [inline doc/system_health_monitoring/system-health-HLD.md:125]: could --> shall

File: doc/system_health_monitoring/system-health-HLD.md [hunk 3]
```diff
@@ -223,11 +282,19 @@ See open question 2 for adding configuration CLIs.
...
+
+1. How to determine the SONiC system is in boot up stage? The current design is to compare the system up time with a "boot_timeout" value. The system up time is got from "cat /proc/uptime". The default "boot_timeout" is 300 seconds and can be configured by configuration. System health service will not do any check until SONiC system finish booting.
+
+```json
+{
+  "boot_timeout": 300
+}
+```
+
+2. Is it necessary to add a CLI to manipulate the configuration file? Like add/remove service/device from ignore list? This is not support for now.
```
Comment (keboliu) [inline doc/system_health_monitoring/system-health-HLD.md:299]: remove the second item, we don't want to support this CLI

File: doc/system_health_monitoring/system-health-HLD.md (hunk 1 above)
Comment (Junchao-Mellanox) [reply doc/system_health_monitoring/system-health-HLD.md:181]: sure

File: doc/system_health_monitoring/system-health-HLD.md (hunk 2 above)
Comment (Junchao-Mellanox) [reply doc/system_health_monitoring/system-health-HLD.md]: fixed

File: doc/system_health_monitoring/system-health-HLD.md (hunk 3 above)
Comment (Junchao-Mellanox) [reply doc/system_health_monitoring/system-health-HLD.md]: fixed

**Pull request merged**